src/shogun/distance/EuclideanDistance.h

```diff
@@ -174,12 +175,16 @@ class CEuclideanDistance: public CDistance
 	 */
 	virtual CFeatures* replace_lhs(CFeatures* lhs);
 
+	SGMatrix<float64_t> get_distance_matrix();
```

Comment:
You're right.. this is not necessary, but helps with readability and similar practice is followed everywhere in Shogun, so... :)